README.md

```diff
@@ -76,6 +76,21 @@ the `open` state.
 When a fallback function is triggered, it's considered a failure, and the
 fallback function will continue to be executed until the breaker is closed.
 
+Also the fallback function accepts the same parameters as the fire function:
```

Comment:
Nit:
```suggestion
The fallback function accepts the same parameters as the fire function:
```

Reply:
Thank you. PR updated :+1:

Reply:
I believe when we do a release,  the docs are updated as part of that npm script,  so that should take care of it.  I've created an issue for trying to automate the doc creation also,  #461 

We could also see about using release-please for doing releases,  similar to what we are about to do on the js cloudevents SDK library, https://github.com/cloudevents/sdk-javascript/pull/329